spec/ics-24-host-requirements/README.md

```diff
@@ -100,6 +100,16 @@ type submitDatagram = (datagram: Datagram) => void
 
 `submitDatagram` allows relayers to relay IBC datagrams directly to the host chain. Host chains MAY require that the relayer submitting the datagram has an account to pay transaction fees, signs over the datagram in a larger transaction structure, etc - `submitDatagram` MUST define any such packaging required.
 
+### Data availability
+
+For safety (e.g. exactly-once packet delivery), host chains MUST have eventual data availability, such that any key-value pairs in state can be eventually retrieved by relayers.
```

Comment:
No time bound is needed?

Reply:
For exactly-once packet delivery, no (the packet will either be delivered once or timeout)